src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IAnonymousFunctionExpression.cs

```diff
@@ -21,7 +21,7 @@ class Program
 {
     static void Main(string[] args)
     {
-        Action x /*<bind>*/= () => F()/*</bind>*/;
+        /*<bind>*/Action x = () => F();/*</bind>*/
```

Comment:
>F() [](start = 35, length = 3)

Do we have a test that requests IOperation just for the lambda expression. We should have tests like that.

Reply:
done